cmd/inetdata-csvrollup/main.go

```diff
@@ -131,17 +144,19 @@ func inputParser(c <-chan string, outc chan<- OutputKey) {
 
 		// Next key hit
 		if ckey != key {
-			outc <- OutputKey{Key: ckey, Vals: cval}
-			ckey = key
-			cval = []string{}
+			if len(cval) != 0 {
+				outc <- OutputKey{Key: ckey, Vals: cval}
+				ckey = key
```

Comment:
Adding a test case for this as I now think this logic is incorrect but currently untriggered with the examples.

Reply:
Addressed.